src/wally-connector.ts

```diff
@@ -216,7 +281,7 @@ class WallyConnector {
           'Success. You may now close this page and refresh the app.')
       : {};
 
-    if (closeWindow) {
+    if (!this.disableRedirectClose) {
       window.setTimeout(window.close, 1000);
```

Comment:
Q: What do we typically use the timeout for?

Reply:
🤷  minor ux - it's a little bit less jarring imo than just closing immediately, and there might be some last bits of cleanup or functions that need to finish before it closes

thoughts?

Reply:
Nothing. It's just me; whenever I see a timeout, it could potentially be a race and that's why people use it. So I'm naturally just picking that up.